from ...exceptions import SviewError


class ResourceError(SviewError):
    pass
